### PR TITLE
Sound user story applied

### PR DESCRIPTION
**Pull request by rojtekin**
Idle sound from entities now plays diffrent sounds depending on entities and volume based on distance. 
Functionality for enemy hit sound and players sword attack sound created.

### CODE REVIEW

File: core/src/view/Sound.java [hunk 1]
```diff
@@ -1,45 +1,61 @@
...
+        swordSwoosh.play(); // TODO: check if the sounds isnt too loud later
+    }
+
+    public void playPlayerDeathSound(){
+        playerDeath.play();
+    }
 
-        com.badlogic.gdx.audio.Sound sound = Gdx.audio.newSound(Gdx.files.internal("Audio/Mouse-squeek.mp3"));
+    public com.badlogic.gdx.audio.Sound getIdleSound(Entity entity) {
+        com.badlogic.gdx.audio.Sound sound = Gdx.audio.newSound(Gdx.files.internal("Audio/"+ entity.getClass().getSimpleName() +"-idle-noise.mp3"));
```
Comment (pkasinski) [inline core/src/view/Sound.java:51]: Should the directory Audio not be written in lowercase?

**Review by pkasinski — COMMENTED** [review]
Looks good overall. I think you should add documentation.

File: core/src/view/Sound.java (hunk 1 above)
Comment (rojtekin) [reply core/src/view/Sound.java:51]: why

**Review by Suedie — APPROVED** [review]
LGTM

**Review by pkasinski — APPROVED** [review]

**Pull request merged**